src/components/TabItem/TabItem.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { Tab } from '../../types/Tab';
+
+interface TabItemProps {
+  tab: Tab;
+  tabId: string | undefined;
```

Comment:
Better to name this `selectedTabId` and use `null` as default value instead of `undefined` if tab is not selected